OpenRA.Mods.Common/AI/SupportPowerDecision.cs

```diff
@@ -94,7 +94,7 @@ public int GetAttractiveness(IEnumerable<Actor> actors, Player firedBy)
 		/// <summary>Makes up part of a decision, describing how to evaluate a target.</summary>
 		public class Consideration
 		{
-			public enum DecisionMetric { Health, Value, None }
+			public enum DecisionMetric { Health, Value, BaseBuilding, None }
```

Comment:
What I'm trying to do is to make the bots more smarter by using the nukes
to target the base building (construction yard) instead of randomly
targeting a group of buildings based on value or health. I couldn't find
anything that can be used to put more weight during parsing the
considerations to determine the target, if there are any, please point them
out to me or give me an exmaple for the yaml.

On Thu, Nov 12, 2015 at 6:04 PM, Zimmermann Gyula <notifications@github.com>
wrote:

> In OpenRA.Mods.Common/AI/SupportPowerDecision.cs
> <https://github.com/OpenRA/OpenRA/pull/9936#discussion_r44675079>:
>
> > @@ -94,7 +94,7 @@ public int GetAttractiveness(IEnumerable<Actor> actors, Player firedBy)
> >  		/// <summary>Makes up part of a decision, describing how to evaluate a target.</summary>
> >  		public class Consideration
> >  		{
> > -			public enum DecisionMetric { Health, Value, None }
> > +			public enum DecisionMetric { Health, Value, BaseBuilding, None }
>
> The Targetable trait is what you're looking for. It does not have a
> hardcoded list, so you can copy ^Building's from defaults.yaml into FACT
> and add your, say Conyard entry into the end of it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenRA/OpenRA/pull/9936/files#r44675079>.
>



-- 
Best Regards,
Fady Anwar